js/components/operatorSignIn/attestation.tsx

```diff
@@ -1,37 +1,60 @@
-import { findEmployeeByBadge, useEmployees } from "../../hooks/useEmployees";
+import { ApiResult } from "../../api";
+import { findEmployeeByBadge } from "../../hooks/useEmployees";
+import { Employee } from "../../models/employee";
+import { className } from "../../util/dom";
 import { useSignInText } from "./text";
-import { ReactElement } from "react";
+import { ReactElement, useEffect, useState } from "react";
 
 export const Attestation = ({
   badge,
+  employees,
   onComplete,
+  loading,
 }: {
   badge: string;
+  employees: ApiResult<Employee[]>;
   onComplete: () => void;
+  loading: boolean;
 }): ReactElement => {
-  const employees = useEmployees();
+  const [entered, setEntered] = useState<string>("");
+  const ready = entered === badge;
+
   if (employees.status === "loading") {
     return <div>Loading...</div>;
   } else if (employees.status === "error") {
-    return <div>Unable to download employee data. Please try again.</div>;
+    return (
+      <div>
+        Unable to download employee data. Please <a href="/">reload</a> to try
```

Comment:
**issue (non-blocking):** I don't think this is super important right now, but I just want to flag that since the `"/"` path is hardcoded here, this will potentially behave in an undesired way if this component is ever rendered on a page with a different path. Maybe that's easy to change quickly but if not don't worry too much about it.

Reply:
Nah, you're right- might as well not make things get weird later! 6a78e9d